addons/web_editor/static/src/js/editor/snippets.options.js

```diff
@@ -5927,10 +5927,8 @@ registry.SnippetMove = SnippetOptionWidget.extend(ColumnLayoutMixin, {
             const hidden = elTop < 0 || bottomHidden;
             if (hidden) {
                 scrollTo(this.$target[0], {
-                    extraOffset: 50,
-                    forcedOffset: bottomHidden ? heightDiff - 50 : undefined,
```

Comment:
Why we ignored the forcedOffset here? It recently added by the [PR]( https://github.com/odoo/odoo/pull/108831/files) so removing it should be breaking something.

Reply:
https://www.awesomescreenshot.com/video/31366815?key=3c389504c1efa4643df296d8c975b00b

May be this is what its breaking. :smiling_face_with_tear: